collectors/etc/elasticsearch_conf.py

```diff
@@ -16,7 +16,7 @@ def get_servers():
   """Get the ElasticSearch servers on this host.
 
   Returns:
-    An iterable of tuples of (host, port)
+    An iterable of tuples of (host, port, user, password)
   """
-  return [ ("localhost", 9200) ]
+  return [ ("localhost", 9200, "user","password") ]
```

Comment:
Default elasticsearch runs on localhost:9200 without any username password.
We should revert this line to original value (but retain the comment)